gadgets/Makefile

```diff
@@ -46,6 +47,13 @@ push: build
 		sudo -E IG_EXPERIMENTAL=true $(IG) image push $(GADGET_REPOSITORY)/$$GADGET:$(GADGET_TAG) || exit 1 ; \
 	done
 
+sign: push
+	@echo "Signing all gadgets"
+	for GADGET in $(GADGETS); do \
+		digest=$$(sudo -E IG_EXPERIMENTAL=true $(IG) image list --no-trunc | grep "$$GADGET " | awk '{ print $$3 }') ; \
+		cosign sign --key env://COSIGN_PRIVATE_KEY --yes --recursive $(GADGET_REPOSITORY)/$$GADGET@$$digest || exit 1 ; \
```

Comment:
Is there any way to avoid having to introduce the key's password N times when running this command?